packages/google-cloud-quotas/.OwlBot.yaml

```diff
@@ -14,5 +14,5 @@
 
 deep-copy-regex:
     - source: /google/api/cloudquotas/(v.*)/.*-py
```

Comment:
should this be (similar to what we have in other packages):

```suggestion
    - source: /google/cloud/quotas/(v.*)/.*-py
```

Reply:
No, `google/api/cloudquotas` is the location in googleapis-gen which will remain unchanged.

https://github.com/googleapis/googleapis-gen/tree/master/google/api/cloudquotas/v1